.evg.yml

```
@@ -230,6 +230,39 @@ tasks:
       - func: "publish maven"
       - func: "trace artifacts"
 
+  # Publishes the release artifacts to S3
+  - name: publish-s3
+    allowed_requesters: ["ad_hoc", "github_tag", "patch"]
+    depends_on:
+      - name: "build"
+        variant: "release"
+      - name: "test-smoke"
+        variant: "release"
+    commands:
+      - *assume_role_cmd
+      - command: s3.get
+        params:
+          working_dir: mongo-jdbc-driver
+          <<: *evg_bucket_config
+          local_file: mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
+          remote_file: ${S3_ARTIFACTS_DIR}/mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
+      - command: s3.get
+        params:
+          working_dir: mongo-jdbc-driver
+          <<: *evg_bucket_config
+          local_file: mongodb-jdbc-eap-${MDBJDBC_VER}-all.jar
+          remote_file: ${S3_ARTIFACTS_DIR}/mongodb-jdbc-eap-${MDBJDBC_VER}.jar
+      - command: s3.put
+        params:
+          aws_key: ${release_aws_key}
+          aws_secret: ${release_aws_secret}
+          local_files_include_filter:
+            - mongodb-jdbc-eap-*.jar
+          remote_file: eap/mongo-jdbc-driver-eap/
+          bucket: translators-connectors-releases
+          permissions: public-read
+          content_type: application/json
```

```suggestion
          content_type: application/java-archive
```
This would be a more appropriate content type for the jar.

---

You are totally right. Copy-pasting....